pages/Manuscripts.xml

```diff
@@ -69,6 +69,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <p>It can be that you find out the same manuscript is present under 2 identifiers.</p>
             <p>You can fix this by merging the two records in one and deleting the one with less information. Remember when doing this to check for references to the id which will go missing, and update them to point to the other identifier.</p>
             </div>
+         <div type="level2">
+            <head>Differenciating between various levels of description</head>
+            <p>Following  <ref target="https://github.com/BetaMasaheft/Documentation/issues/2700"
+               >issue 2700</ref>, to mark a rudimentary description as "stub" (if the cataloguer wants others consider it as "stub"), one needs to 
+               write the word "stub" anywhere in the change history.  When the record has stopped being a stub as it has been developed further then in the first change line the word stub must replaced with something like initial version or anything else.  
```

Comment:
I think it would be nice to include as an example the complete `<revisionDesc>` of a file that followed this process (as two `<egXML xmlns="http://www.tei-c.org/ns/Examples"> `elements). The original one using "stub" could be included after "one needs to write the word "stub" anywhere in the change history.", and the revised one after "initial version or anything else". ANLcr1 is a real example that comes to my mind just because I was involved, any other file would work as well.